pages/Configuring/Advanced-config.md

```diff
@@ -304,6 +304,8 @@ You have six choices:
 
 - Name: e.g. `name:Web`, `name:Anime` or `name:Better anime`
 
+- Previous workspace: `previous`
```

Comment:
umm, make the amount of choices to "seven" above lol

Reply:
Ugh must've been tired last night, soz.